rust/operator/src/kerberos.rs

```diff
@@ -1,9 +1,21 @@
 use stackable_hdfs_crd::{
-    constants::{HADOOP_SECURITY_AUTHENTICATION, SSL_CLIENT_XML, SSL_SERVER_XML},
-    HdfsCluster, HdfsRole,
+    constants::{SSL_CLIENT_XML, SSL_SERVER_XML},
+    HdfsCluster,
 };
+use stackable_operator::commons::product_image_selection::ResolvedProductImage;
 
-use crate::config::{CoreSiteConfigBuilder, HdfsSiteConfigBuilder};
+use crate::{
+    config::{CoreSiteConfigBuilder, HdfsSiteConfigBuilder},
+    hdfs_controller::Error,
+};
+
+pub fn check_if_supported(resolved_product_image: &ResolvedProductImage) -> Result<(), Error> {
+    if resolved_product_image.product_version.starts_with("3.2.") {
```

Comment:
It feels hacky but I guess it's simple enough and works... Especially since we can drop it when we drop 3.2.x support.